classes/student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age:, classroom:, name: 'Unknow', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] For the super method, take note to arrange your arguments based on how the Person constructor is defined. The arrangement is in this order `age`, `name`, `parent_permission`.

- [ ] Since you are using the colon notation to define your parameters, you need to define it properly in the super method. Should be `name: name`.

  ```
  def initialize(age:, classroom:, name: 'Unknow', parent_permission: true)
    super(age, name: name, parent_permission: parent_permission)
    ...
  ```

Reply:
Done